rollouts/api/v1alpha1/remotersync_types.go

```diff
@@ -59,10 +71,11 @@ type GitInfo struct {
 type Metadata struct {
 	Labels      map[string]string `json:"labels,omitempty"`
 	Annotations map[string]string `json:"annotations,omitempty"`
+	Namespace   string            `json:"namespace,omitempty"`
```

Comment:
I'm going to update this to take the Namespace from the Rollout object directly, instead of making it configurable